docs/preview/features/powershell/datafactory.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Nitpick suggestion but maybe we should use `azure-data-factory.md` for filenames since those are part of the URL?

Reply:
Yes, you are right, as always!

Reply:
Yes, changed.